packages/components/src/buttons/src/Button.module.css

```diff
@@ -1,7 +1,11 @@
 .hop-Button {
     --hop-Button-padding: 0 var(--hop-space-inset-md);
     --hop-Button-border-radius: var(--hop-shape-rounded-md);
-    --hop-Button-focus-ring-color: var(--hop-primary-surface-hover); /* TODO: place to centralize tokens like this? */
+    --hop-Button-focus-ring-color: var(--hop-primary-border-focus); /* TODO: replace with a token */
+
+    /* sizes */
+    --hop-Button-icon-only-sm-padding: 0 var(--hop-space-inset-xs);
```

Comment:
Having a padding shorthand in a component token could have some unwanted effects. A user who only needs the x or y value won't be able to refer to these.

We could do it like so: 

```
--hop-Button-icon-only-sm-padding-x: var(--hop-space-inset-xs);
--hop-Button-icon-only-sm-padding-y: 0;
```